modules/t/21-query-val.t

```diff
@@ -25,7 +25,7 @@ ok( defined $srv, "Service found" );
 
 my $mart;
 for my $m ( @{ $srv->get_marts() } ) {
-	if ( $m->name() =~ m/fungi_mart_[0-9]+/ ) {
+	if ( $m->name() =~ m/fungi_mart+/ ) {
```

Comment:
Not sure if that's intended but that regex would match strings ending with one or more occurrences of `t` (e.g. `fungi_marttttt`). Probably better to change into: 

```suggestion
	if ( $m->name() =~ m/fungi_mart/ ) {
```